src/AccessibilityInsights.Extensions.AzureDevOps/FileIssue/IssueFileForm.cs

```diff
@@ -110,8 +113,21 @@ private State updateState(bool revert = false)
                     case State.Initializing:
                         if (currentUri.Host == Url.Host && currentUri.AbsolutePath == Url.AbsolutePath)
                             _currentState = State.TemplateIsOpen;
+                        else
+                        {
```

Comment:
having braces around line 118 but not 115 seems unbalanced, somehow. Given the pattern used in this file, maybe remove the braces around line 118?